### PR TITLE
Improve error messages in the quarks repo

### PR DESCRIPTION
**Pull request by rohitsakala**
[#165014759](https://www.pivotaltracker.com/n/projects/2192232/stories/165014759)

### CODE REVIEW

File: cmd/internal/bpm_configs.go [hunk 1]
```diff
@@ -45,56 +45,57 @@ instance group.
 
 		log = newLogger()
 		defer log.Sync()
+
 		boshManifestPath := viper.GetString("bosh-manifest-path")
 		if len(boshManifestPath) == 0 {
-			return fmt.Errorf("manifest cannot be empty")
+			return errors.Errorf("%s %s", "bpm-configs command failed.", "bosh-manifest-path flag is empty.")
```
Comment (edwardstudy) [inline cmd/internal/bpm_configs.go]: constant `"bpm-configs command failed."`, `"data-gather command failed."`, etc?

Comment (manno) [reply cmd/internal/bpm_configs.go]:
```suggestion
			return errors.Errorf("bpm-configs command failed. bosh-manifest-path flag is empty.")
```

Comment (rohitsakala) [reply cmd/internal/bpm_configs.go]: Gotcha. :+1:

**Comment by manno** [comment]
I solved most of the conflicts here

https://github.com/cloudfoundry-incubator/cf-operator/pull/new/mm/rohit-errors-rebased

**Pull request merged**